recipes-bsp/tegra-binaries/tegra-helper-scripts/make-sdcard.sh

```diff
@@ -334,6 +334,7 @@ if ! sgdisk "$output" --verify >/dev/null 2>&1; then
     echo "ERR: verification failed for $output" >&2
     exit 1
 fi
+sleep 1
```

Comment:
A minor thing, but could you move this inside the `if` block below, so it only happens if we're going to do a `partprobe`?

Reply:
Good idea-- done.

Reply:
Thanks!